vlan_and_lagg.rst Adjustments (#613)
opnsense/docs: source/manual/how-tos/vlan_and_lagg.rst

    The trunk from `Managed Switch` to `Access Point` has to be configured in a mixed mode with an untagged (default) VLAN and a tagged VLAN.
    This is in contrast to the trunk that is connected to the OPNsense, which has no untagged (default) VLAN.

.. Tip::

>     If we on the switch specify a VLAN in the Access Port mode, the switch itself will on Ingress of this port assign any unTAGGed frame to the specific

Reviewer: ```suggestion
    The `Access Port` of a `Managed Switch` will tag ingress frames with its configured VLAN, and strip egress frames of VLAN tags.
```

>     VLAN. And on Egress of this port remove the VLAN TAG.

Reviewer: ```suggestion
```

.. Tip::

>     It is good practice to allow/configure on switch and its ports only those VLANs that are needed and used for that specific Port or LAGG. 

Reviewer: ```suggestion
    It is good practice to configure the ports of a `Managed Switch` only with VLANs that are needed for that specific port or LAGG. 
```

>     This is called manual VLAN Pruning.

Author: Those tips, 

1st tip - the  Ingress Egress unTAGGed doesn't reexplain what is already there just it advances on it, how from the perspective of the VLAN the switch is handling those. 

2nd tip - Actually by the way how you wrote the docs you already enforce VLAN Pruning, because when you specify in the guideline that only these VLANs should be on the TRUNK this is VLAN Pruning. I only pointed it out and called it its true name.

    the UPLINK and provides re-convergence in case one of the ports in the LAGG bundle goes down.
    LAGG Proto needs to be set on both devices connecting the LAGG at the same value.   

.. Attention::

    Fast timeout, LACP timeout has two modes; Slow/normal and Fast. It handles how fast the re-convergence occurs in case of link failure. 
    This specifies how often are the heartbeats sent between the two LAGG connected devices.
    The Slow/normal (most cases default), will sent heartbeats each 30s. The Fast, will sent heartbeats each 1s.
    In most scenarios keeping the LACP timeout on Slow/normal is preferable, due to vague vendor implementation of the LACP fast timeout it can cause connectivity disruption.
    LACP timeout needs to be set on both devices connecting the LAGG at the same value, if not, heartbeats can be missed which will cause connectivity disruption.

.. Note::

    Hash Layers, specifies how the device will loadbalance the traffic across physical links in the LAGG bundle.
    This is done per the 5-tuple if the LACP device implementation allows it. 

>     Hash Layers, do not need to be the same between two connecting devices, it can be considered as a best practice but its not a rule of must be.   

Reviewer: I wanted to keep the LAGG portion short here since it is optional and the document is more focused on VLANs.

I think dropping the MLAG note was a good idea, but adding that much instead muddies the focus too. Maybe these are better in the linked parent document. https://docs.opnsense.org/manual/other-interfaces.html#lagg

The recommended values in the the table should be enough here in my opinion.

.. Note::

     Only routed traffic can be filtered by a central firewall. Devices in the same VLAN communicate directly by using ARP or NDP to discover their neighbors.

>      Such devices will not hit the Firewall/Router and you will not see any traffic on the dedicated GW device.

Author: Hm it was not my intent to create confusion here. The sentence it self is the follow up on _"Devices in the same VLAN communicate directly"_

If its creating a possible confusing I agree it should be removed.
